config/dispatcher/dispatcher.ex

```diff
@@ -10,8 +10,14 @@ defmodule Dispatcher do
 ###############################################################################
 # Resources
 ###############################################################################
+  get "/accounts/*path", %{ accept: %{ json: true } } do
+    forward conn, path, "http://resource/accounts/"
+  end
+
+  get "/complaint-forms/*path", %{ accept: %{ json: true } } do
```

Comment:
We need to shield this somehow so only logged in users can fetch submitted complaint forms.